src/utils/rootkeypackage_utils/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required (VERSION 3.5)
+
+project (rootkeypackage_utils)
+
+compileasc99 ()
+
+find_package (azure_c_shared_utility REQUIRED)
+
+add_library (${PROJECT_NAME} STATIC src/rootkeypackage_utils.c)
+add_library (aduc::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
+
+target_include_directories (${PROJECT_NAME} PUBLIC inc ${ADUC_EXPORT_INCLUDES})
+
+target_link_libraries (${PROJECT_NAME}
```

Comment:
Don't we need to include the library for the result.h file?

Reply:
result.h is in src/inc/aduc/result.h so handled by
target_include_directories (${target_name} PUBLIC inc ${ADUC_EXPORT_INCLUDES})
I may be missing something